web/forms.py

```
@@ -1677,3 +1677,19 @@ class Meta:
         fields = ["name", "description", "course", "max_members", "is_private"]
+
+
+class NFTBadgeForm(forms.ModelForm):
+    class Meta:
+        model = Achievement
+        fields = ["title", "description", "achievement_type", "badge_icon", "course"]
+        widgets = {
+            "course": forms.Select(attrs={"class": "form-select"}),
+        }
+
+    def save(self, commit=True):
```

Ruff (0.8.2)

Missing return type annotation for private function `save`

(ANN202)